x/ibc/05-port/keeper/keeper.go

```diff
@@ -45,6 +47,8 @@ func (k *Keeper) BindPort(ctx sdk.Context, portID string) capability.Capability
 		panic(err.Error())
 	}
 
+	k.router[portID] = cbs
```

Comment:
The problem is that `BindPort` will not be re-called when e.g. `gaiad` is stopped & re-started

Reply:
Can we delete this now?